homeassistant/components/bbox/sensor.py

```diff
@@ -126,6 +128,12 @@ def update(self):
             self._state = round(self.bbox_data.data["rx"]["bandwidth"] / 1000, 2)
         elif self.type == "current_up_bandwidth":
             self._state = round(self.bbox_data.data["tx"]["bandwidth"] / 1000, 2)
+        elif self.type == "uptime":
+            self._state = str(
+                timedelta(seconds=self.bbox_data.router_infos["device"]["uptime"])
```

Comment:
We should not put relative times in the state machine since it will update a lot and just spams updates due to the time that is passing.

Please use a timestamp device class for this specific sensor as a device class and calculate/format the time to absolute UTC iso format.

Example from other integration that calculates this: <https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/wled/sensor.py#L121>

Reply:
I hope it's better this time, sorry for my lack of knowledge 😄